modules/ROOT/pages/workday/workday-connector-studio.adoc

```diff
@@ -53,11 +53,17 @@ The required fields for the On New Objects input source are:
 |Studio Field |Description
 |Connector configuration |Global Element Properties configuration name
 |Object type |One of: `ABSENCE_INPUTS`, `ACADEMIC_CONTACTS`, `ALL_CURRENCIES`, `APPLICANTS`, `BASIC_CUSTOMERS`, `CANDIDATES`, `CERTIFICATION_ISSUERS`, `CERTIFICATIONS`, `COMPETENCIES`, `DEGREES`, `DONORS`, `EDUCATION_TEST_RESULTS`, `EDUCATIONAL_INSTITUTION_COURSE`, `HEALTH_CARE_RATES`, `INVESTORS`, `ORGANIZATIONS`, `PAYMENTS`, `PROGRAMS_OF_STUDY`, `STUDENTS`, `STUDENTS_APPLICATIONS`, `WORKERS`
+|Scheduling Strategy |Configures the scheduler that triggers polling *&#8224;*
 |Username |The username to log in with
 |Password |The corresponding password
-|Tenant Name |The Workday Tenant ID, which usually has the suffix `pt_1` appended to it, such as `acme_pt1`
+|Tenant Name |The Workday tenant ID, which usually has the suffix `_pt1` appended to it, such as `acme_pt1`
 |===
 
+*&#8224;* *Scheduling Strategy* is used to to tell the source how it can poll the data. This field accepts two options:
```

Comment:
minor active voice suggestion:
```suggestion
*&#8224;* *Scheduling Strategy* tells the source how it can poll the data. This field accepts two options:
```